src/parameterizations/vertical/MOM_set_viscosity.F90

```diff
@@ -1995,11 +1995,13 @@ subroutine set_visc_init(Time, G, GV, US, param_file, diag, visc, CS, restart_CS
   real    :: TKE_decay_dflt  ! The default value of a coefficient scaling the vertical decay
                              ! rate of TKE [nondim]
   real    :: bulk_Ri_ML_dflt ! The default bulk Richardson number for a bulk mixed layer [nondim]
-  real    :: Kv_background   ! The background kinematic viscosity in the interior [m2 s-1]
+  real    :: Kv_background   ! The background kinematic viscosity in the interior [Z2 T-1 ~> m2 s-1]
   real    :: omega_frac_dflt ! The default value for the fraction of the absolute rotation rate that
                              ! is used in place of the absolute value of the local Coriolis
                              ! parameter in the denominator of some expressions [nondim]
-  real    :: Chan_max_thick_dflt ! The default value for CHANNEL_DRAG_MAX_THICK [m]
+  real    :: Chan_max_thick_dflt ! The default value for CHANNEL_DRAG_MAX_THICK [Z ~> m]
+  real    :: Hbbl            ! The static bottom boundary layer thickness [Z ~> m].
```

Comment:
Hbbl and BBL_thick_min are now used directly instead of CS%Hbbl and CS%BBL_thick_min

Reply:
Yes, your description is correct. They are being treated as distinct variables because the local copies have units of [Z ~> m], whereas the ones in the control structure have units of [H ~> m or kg m-2], in large part because you can not be guaranteed to be able to invert something in H units to exactly recover a default value in [m] in non-Boussinesq mode due to the multiplication and division by Rho_0 that this would imply.  Conversion back and forth between [Z ~> m] and [m] is exact due to the integer-power-of-2 scaling that is used.